multi-agent/lib/mas/elements/nodes/claude_agent/card_builder.py

```
@@ -0,0 +1,63 @@
+from mas.elements.common.card.interface import CardBuilder
+
+
+class ClaudeAgentCardBuilder(CardBuilder):
+    """
+    Builds element card for Claude Agent Node.
+
+    Skills = allowed tools (Read, Write, Bash, etc.)
+    Capabilities = model, permission mode, max turns
+    """
+
+    def build(self, input: CardBuildInput) -> ElementCard:
```

Ruff (0.15.15)

[error] Function argument `input` is shadowing a Python builtin

(A002)